src/runtime/symbols.ts

```diff
@@ -8,5 +8,7 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-// tslint:disable-next-line: variable-name
-export const Symbols = {identifier: Symbol('id')};
+// TypeScript seems to lose the necessary type info if this symbol is wrapped in an object and then
+// used as an interface key (e.g. 'interface Foo { [Symbols.internals]: {...} }'), so we just have
+// to export it as a standard variable.
```

Comment:
Could you also add a comment explaining what the purpose of this symbol is? It's good to explain why it can't be nested inside an object, but we still don't know what it's for..

Reply:
I covered that in the comment added to EntityInternals, which I think is the more relevant place for the it.

Reply:
Could you add something like "See the EntityInternals class for details", or something like that? This file doesn't really have any info on what these symbols are supposed to be for.

Reply:
Done.